2018-06-28 03:20:21.285 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 03:20:21.288 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 03:21:21.285 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 03:21:21.289 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 03:22:21.285 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 03:22:21.288 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 03:22:27.263 19219 DEBUG oslo_concurrency.lockutils [req-42b5cb1f-9506-4819-9dc9-c7a5fed00e8c 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:22:27.924 19219 DEBUG neutron.agent.dhcp.agent [req-e2dcdf55-0291-44ae-b247-2ecc709aab19 - - - - -] Calling driver for network: 13c94b81-58d3-4646-b469-39c74e1b872e action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-28 03:22:27.926 19219 DEBUG neutron.agent.linux.utils [req-e2dcdf55-0291-44ae-b247-2ecc709aab19 - - - - -] Unable to access /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:22:27.928 19219 DEBUG neutron.agent.linux.dhcp [req-e2dcdf55-0291-44ae-b247-2ecc709aab19 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-13c94b81-58d3-4646-b469-39c74e1b872e on network 13c94b81-58d3-4646-b469-39c74e1b872e does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1267
2018-06-28 03:22:27.929 19219 DEBUG neutron.agent.linux.dhcp [req-e2dcdf55-0291-44ae-b247-2ecc709aab19 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-13c94b81-58d3-4646-b469-39c74e1b872e on network 13c94b81-58d3-4646-b469-39c74e1b872e does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1288
2018-06-28 03:22:31.298 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.315 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.328 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.344 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.359 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.374 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', 'link', 'set', 'tap4e7d3607-d1', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.398 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-o', 'link', 'show', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.413 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-28 03:22:31.438 19219 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tap4e7d3607-d1) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-28 03:22:31.439 19219 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-06-28 03:22:31.440 19219 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tap4e7d3607-d1) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-28 03:22:31.442 19219 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'4e7d3607-d1d8-4655-9834-fd173e76fb95', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:b1:e5:6b'})), record=tap4e7d3607-d1) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-28 03:22:31.472 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap4e7d3607-d1', 'address', 'fa:16:3e:b1:e5:6b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.485 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.496 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap4e7d3607-d1', 'netns', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.519 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', 'link', 'set', 'tap4e7d3607-d1', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.532 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', 'link', 'set', 'tap4e7d3607-d1', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.547 19219 DEBUG oslo_concurrency.lockutils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Acquired semaphore "iptables-qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:22:31.549 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.578 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.598 19219 DEBUG neutron.agent.linux.iptables_manager [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-06-28 03:22:31.600 19219 DEBUG oslo_concurrency.lockutils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Releasing semaphore "iptables-qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:22:31.602 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-o', 'link', 'show', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.615 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', 'addr', 'show', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.630 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-4', 'addr', 'add', '11.235.1.2/24', 'scope', 'global', 'dev', 'tap4e7d3607-d1', 'brd', '11.235.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.642 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-4', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.658 19219 DEBUG neutron.agent.linux.dhcp [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Setting IPv4 gateway for dhcp netns on net 13c94b81-58d3-4646-b469-39c74e1b872e to 11.235.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1164
2018-06-28 03:22:31.659 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.235.1.1', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.674 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-6', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.687 19219 DEBUG neutron.agent.linux.dhcp [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Cleaning stale devices for network 13c94b81-58d3-4646-b469-39c74e1b872e _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1369
2018-06-28 03:22:31.688 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.700 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.723 19219 DEBUG neutron.agent.linux.dhcp [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Building initial lease file: /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:623
2018-06-28 03:22:31.726 19219 DEBUG neutron.agent.linux.dhcp [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Done building initial lease file /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/leases with contents:
1530235351 fa:16:3e:b1:e5:6b 11.235.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:651
2018-06-28 03:22:31.727 19219 DEBUG neutron.agent.linux.dhcp [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Building host file: /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-28 03:22:31.728 19219 DEBUG neutron.agent.linux.dhcp [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Done building host file /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-28 03:22:31.732 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Unable to access /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:22:31.734 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host', '--addn-hosts=/var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tap4e7d3607-d1', '--dhcp-range=set:tag0,11.235.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:31.764 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:22:31.765 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:22:31.768 19219 DEBUG neutron.agent.linux.external_process [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] No process started for 13c94b81-58d3-4646-b469-39c74e1b872e disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-28 03:22:31.769 19219 DEBUG oslo_concurrency.lockutils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:22:31.771 19219 DEBUG oslo_concurrency.lockutils [req-42b5cb1f-9506-4819-9dc9-c7a5fed00e8c 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:22:32.237 19219 DEBUG neutron.agent.dhcp.agent [req-5a705e7d-34c5-45eb-a22a-56eaabb0a2a4 - - - - -] Calling driver for network: 13c94b81-58d3-4646-b469-39c74e1b872e action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-28 03:22:32.240 19219 DEBUG neutron.agent.linux.dhcp [req-5a705e7d-34c5-45eb-a22a-56eaabb0a2a4 - - - - -] Building host file: /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-28 03:22:32.244 19219 DEBUG neutron.agent.linux.dhcp [req-5a705e7d-34c5-45eb-a22a-56eaabb0a2a4 - - - - -] Done building host file /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-28 03:22:32.248 19219 DEBUG neutron.agent.linux.utils [req-5a705e7d-34c5-45eb-a22a-56eaabb0a2a4 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27944'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:32.264 19219 DEBUG neutron.agent.linux.dhcp [req-5a705e7d-34c5-45eb-a22a-56eaabb0a2a4 - - - - -] Reloading allocations for network: 13c94b81-58d3-4646-b469-39c74e1b872e reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-28 03:22:32.266 19219 DEBUG neutron.agent.linux.utils [req-5a705e7d-34c5-45eb-a22a-56eaabb0a2a4 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-4', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:32.279 19219 DEBUG neutron.agent.linux.utils [req-5a705e7d-34c5-45eb-a22a-56eaabb0a2a4 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-6', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:32.293 19219 DEBUG neutron.agent.linux.utils [req-5a705e7d-34c5-45eb-a22a-56eaabb0a2a4 - - - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:22:32.294 19219 DEBUG neutron.agent.linux.utils [req-5a705e7d-34c5-45eb-a22a-56eaabb0a2a4 - - - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:22:32.295 19219 DEBUG neutron.agent.linux.external_process [req-5a705e7d-34c5-45eb-a22a-56eaabb0a2a4 - - - - -] No process started for 13c94b81-58d3-4646-b469-39c74e1b872e disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-28 03:22:32.295 19219 DEBUG oslo_concurrency.lockutils [req-5a705e7d-34c5-45eb-a22a-56eaabb0a2a4 - - - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:22:32.297 19219 DEBUG oslo_concurrency.lockutils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:22:32.297 19219 DEBUG neutron.agent.dhcp.agent [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-28T01:22:28Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-13c94b81-58d3-4646-b469-39c74e1b872e, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6ed2183a-68cb-4948-a7f8-adef908f6ca6', u'ip_address': u'11.235.1.2'}], id=4e7d3607-d1d8-4655-9834-fd173e76fb95, mac_address=fa:16:3e:b1:e5:6b, name=, network_id=13c94b81-58d3-4646-b469-39c74e1b872e, port_security_enabled=False, project_id=d1812a4a358f413d963f4743db11b81f, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=d1812a4a358f413d963f4743db11b81f, updated_at=2018-06-28T01:22:29Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:409
2018-06-28 03:22:32.298 19219 DEBUG oslo_concurrency.lockutils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:22:32.299 19219 DEBUG oslo_concurrency.lockutils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:22:32.300 19219 INFO neutron.agent.dhcp.agent [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-28T01:22:28Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-13c94b81-58d3-4646-b469-39c74e1b872e, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6ed2183a-68cb-4948-a7f8-adef908f6ca6', u'ip_address': u'11.235.1.2'}], id=4e7d3607-d1d8-4655-9834-fd173e76fb95, mac_address=fa:16:3e:b1:e5:6b, name=, network_id=13c94b81-58d3-4646-b469-39c74e1b872e, port_security_enabled=False, project_id=d1812a4a358f413d963f4743db11b81f, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=d1812a4a358f413d963f4743db11b81f, updated_at=2018-06-28T01:22:30Z
2018-06-28 03:22:32.301 19219 DEBUG neutron.agent.dhcp.agent [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Calling driver for network: 13c94b81-58d3-4646-b469-39c74e1b872e action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-28 03:22:32.302 19219 DEBUG neutron.agent.linux.dhcp [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Building host file: /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-28 03:22:32.305 19219 DEBUG neutron.agent.linux.dhcp [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Done building host file /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-28 03:22:32.310 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27944'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:32.323 19219 DEBUG neutron.agent.linux.dhcp [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Reloading allocations for network: 13c94b81-58d3-4646-b469-39c74e1b872e reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-28 03:22:32.325 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-4', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:32.339 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-6', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:22:32.351 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:22:32.352 19219 DEBUG neutron.agent.linux.utils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:22:32.353 19219 DEBUG neutron.agent.linux.external_process [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] No process started for 13c94b81-58d3-4646-b469-39c74e1b872e disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-28 03:22:32.355 19219 DEBUG oslo_concurrency.lockutils [req-cdda662c-4bb1-46e0-beb1-a223344d4c3b - - - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:23:05.328 19219 DEBUG oslo_concurrency.lockutils [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:23:05.329 19219 INFO neutron.agent.dhcp.agent [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-28T01:23:03Z, description=, device_id=5c2695e1-e226-49f3-942b-1eb09cd71c2b, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6ed2183a-68cb-4948-a7f8-adef908f6ca6', u'ip_address': u'11.235.1.1'}], id=a8c56f71-d50a-47c6-b375-1e561fca6769, mac_address=fa:16:3e:43:84:f7, name=, network_id=13c94b81-58d3-4646-b469-39c74e1b872e, port_security_enabled=False, project_id=d1812a4a358f413d963f4743db11b81f, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=d1812a4a358f413d963f4743db11b81f, updated_at=2018-06-28T01:23:04Z
2018-06-28 03:23:05.331 19219 DEBUG neutron.agent.dhcp.agent [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Calling driver for network: 13c94b81-58d3-4646-b469-39c74e1b872e action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-28 03:23:05.332 19219 DEBUG neutron.agent.linux.dhcp [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Building host file: /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-28 03:23:05.334 19219 DEBUG neutron.agent.linux.dhcp [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Done building host file /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-28 03:23:05.339 19219 DEBUG neutron.agent.linux.utils [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27944'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:05.381 19219 DEBUG neutron.agent.linux.dhcp [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Reloading allocations for network: 13c94b81-58d3-4646-b469-39c74e1b872e reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-28 03:23:05.389 19219 DEBUG neutron.agent.linux.utils [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-4', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:05.412 19219 DEBUG neutron.agent.linux.utils [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-6', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:05.439 19219 DEBUG neutron.agent.linux.utils [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:23:05.442 19219 DEBUG neutron.agent.linux.utils [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:23:05.443 19219 DEBUG neutron.agent.linux.external_process [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] No process started for 13c94b81-58d3-4646-b469-39c74e1b872e disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-28 03:23:05.444 19219 DEBUG oslo_concurrency.lockutils [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:23:07.115 19219 DEBUG oslo_concurrency.lockutils [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:23:07.116 19219 INFO neutron.agent.dhcp.agent [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-28T01:23:03Z, description=, device_id=5c2695e1-e226-49f3-942b-1eb09cd71c2b, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6ed2183a-68cb-4948-a7f8-adef908f6ca6', u'ip_address': u'11.235.1.1'}], id=a8c56f71-d50a-47c6-b375-1e561fca6769, mac_address=fa:16:3e:43:84:f7, name=, network_id=13c94b81-58d3-4646-b469-39c74e1b872e, port_security_enabled=False, project_id=d1812a4a358f413d963f4743db11b81f, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=d1812a4a358f413d963f4743db11b81f, updated_at=2018-06-28T01:23:04Z
2018-06-28 03:23:07.118 19219 DEBUG neutron.agent.dhcp.agent [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Calling driver for network: 13c94b81-58d3-4646-b469-39c74e1b872e action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-28 03:23:07.120 19219 DEBUG neutron.agent.linux.dhcp [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Building host file: /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-28 03:23:07.125 19219 DEBUG neutron.agent.linux.dhcp [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Done building host file /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-28 03:23:07.132 19219 DEBUG neutron.agent.linux.utils [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27944'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:07.151 19219 DEBUG neutron.agent.linux.dhcp [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Reloading allocations for network: 13c94b81-58d3-4646-b469-39c74e1b872e reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-28 03:23:07.155 19219 DEBUG neutron.agent.linux.utils [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-4', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:07.177 19219 DEBUG neutron.agent.linux.utils [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-6', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:07.192 19219 DEBUG neutron.agent.linux.utils [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:23:07.193 19219 DEBUG neutron.agent.linux.utils [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:23:07.194 19219 DEBUG neutron.agent.linux.external_process [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] No process started for 13c94b81-58d3-4646-b469-39c74e1b872e disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-28 03:23:07.196 19219 DEBUG oslo_concurrency.lockutils [req-c7ca8562-699a-405f-895d-89e1c5a9f370 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:23:09.631 19219 DEBUG oslo_concurrency.lockutils [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:23:09.632 19219 INFO neutron.agent.dhcp.agent [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-28T01:23:03Z, description=, device_id=5c2695e1-e226-49f3-942b-1eb09cd71c2b, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6ed2183a-68cb-4948-a7f8-adef908f6ca6', u'ip_address': u'11.235.1.1'}], id=a8c56f71-d50a-47c6-b375-1e561fca6769, mac_address=fa:16:3e:43:84:f7, name=, network_id=13c94b81-58d3-4646-b469-39c74e1b872e, port_security_enabled=False, project_id=d1812a4a358f413d963f4743db11b81f, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=d1812a4a358f413d963f4743db11b81f, updated_at=2018-06-28T01:23:06Z
2018-06-28 03:23:09.633 19219 DEBUG neutron.agent.dhcp.agent [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Calling driver for network: 13c94b81-58d3-4646-b469-39c74e1b872e action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-28 03:23:09.635 19219 DEBUG neutron.agent.linux.dhcp [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Building host file: /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-28 03:23:09.637 19219 DEBUG neutron.agent.linux.dhcp [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Done building host file /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-28 03:23:09.641 19219 DEBUG neutron.agent.linux.utils [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27944'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:09.662 19219 DEBUG neutron.agent.linux.dhcp [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Reloading allocations for network: 13c94b81-58d3-4646-b469-39c74e1b872e reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-28 03:23:09.665 19219 DEBUG neutron.agent.linux.utils [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-4', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:09.685 19219 DEBUG neutron.agent.linux.utils [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-6', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:09.702 19219 DEBUG neutron.agent.linux.utils [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:23:09.703 19219 DEBUG neutron.agent.linux.utils [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:23:09.704 19219 DEBUG neutron.agent.linux.external_process [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] No process started for 13c94b81-58d3-4646-b469-39c74e1b872e disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-28 03:23:09.705 19219 DEBUG oslo_concurrency.lockutils [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:23:10.559 19219 DEBUG oslo_concurrency.lockutils [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:23:10.561 19219 INFO neutron.agent.dhcp.agent [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-28T01:23:03Z, description=, device_id=5c2695e1-e226-49f3-942b-1eb09cd71c2b, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6ed2183a-68cb-4948-a7f8-adef908f6ca6', u'ip_address': u'11.235.1.1'}], id=a8c56f71-d50a-47c6-b375-1e561fca6769, mac_address=fa:16:3e:43:84:f7, name=, network_id=13c94b81-58d3-4646-b469-39c74e1b872e, port_security_enabled=False, project_id=d1812a4a358f413d963f4743db11b81f, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=d1812a4a358f413d963f4743db11b81f, updated_at=2018-06-28T01:23:10Z
2018-06-28 03:23:10.563 19219 DEBUG neutron.agent.dhcp.agent [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Calling driver for network: 13c94b81-58d3-4646-b469-39c74e1b872e action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-28 03:23:10.565 19219 DEBUG neutron.agent.linux.dhcp [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Building host file: /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-28 03:23:10.567 19219 DEBUG neutron.agent.linux.dhcp [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Done building host file /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-28 03:23:10.571 19219 DEBUG neutron.agent.linux.utils [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27944'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:10.590 19219 DEBUG neutron.agent.linux.dhcp [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Reloading allocations for network: 13c94b81-58d3-4646-b469-39c74e1b872e reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-28 03:23:10.594 19219 DEBUG neutron.agent.linux.utils [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-4', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:10.616 19219 DEBUG neutron.agent.linux.utils [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-6', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:10.631 19219 DEBUG neutron.agent.linux.utils [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:23:10.632 19219 DEBUG neutron.agent.linux.utils [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:23:10.633 19219 DEBUG neutron.agent.linux.external_process [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] No process started for 13c94b81-58d3-4646-b469-39c74e1b872e disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-28 03:23:10.634 19219 DEBUG oslo_concurrency.lockutils [req-d92cd80f-086f-4cdf-849a-9814eeab2357 - - - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:23:18.623 19219 DEBUG oslo_concurrency.lockutils [req-219ccf7a-e134-4e06-a2da-a907079f7707 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Acquired semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:23:18.624 19219 DEBUG oslo_concurrency.lockutils [req-219ccf7a-e134-4e06-a2da-a907079f7707 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Releasing semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:23:21.215 19219 DEBUG oslo_concurrency.lockutils [req-c791cf86-0f71-4f36-ad7c-3043acc8605e - - - - -] Acquired semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:23:21.216 19219 DEBUG oslo_concurrency.lockutils [req-c791cf86-0f71-4f36-ad7c-3043acc8605e - - - - -] Releasing semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:23:21.286 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 03:23:21.289 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 03:23:22.227 19219 DEBUG oslo_concurrency.lockutils [req-c791cf86-0f71-4f36-ad7c-3043acc8605e - - - - -] Acquired semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:23:22.228 19219 DEBUG oslo_concurrency.lockutils [req-c791cf86-0f71-4f36-ad7c-3043acc8605e - - - - -] Releasing semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:23:41.793 19219 DEBUG oslo_concurrency.lockutils [req-b1b52991-302e-426b-9934-7617a7a8e23d 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:23:41.796 19219 INFO neutron.agent.dhcp.agent [req-b1b52991-302e-426b-9934-7617a7a8e23d 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-28T01:23:40Z, description=, device_id=460cb1b1-3a30-4fd3-b6a1-1297a2ddab08, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6ed2183a-68cb-4948-a7f8-adef908f6ca6', u'ip_address': u'11.235.1.7'}], id=4c7cc62b-ee0a-4d90-bc23-c8ccfd1531e4, mac_address=fa:16:3e:93:7c:1b, name=, network_id=13c94b81-58d3-4646-b469-39c74e1b872e, port_security_enabled=True, project_id=d1812a4a358f413d963f4743db11b81f, revision_number=3, security_groups=[u'ebbd80ef-9432-47b6-b001-b2448e141ec1'], status=DOWN, tags=[], tenant_id=d1812a4a358f413d963f4743db11b81f, updated_at=2018-06-28T01:23:40Z
2018-06-28 03:23:41.800 19219 DEBUG neutron.agent.dhcp.agent [req-b1b52991-302e-426b-9934-7617a7a8e23d 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Calling driver for network: 13c94b81-58d3-4646-b469-39c74e1b872e action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-28 03:23:41.802 19219 DEBUG neutron.agent.linux.dhcp [req-b1b52991-302e-426b-9934-7617a7a8e23d 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Building host file: /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-28 03:23:41.811 19219 DEBUG neutron.agent.linux.dhcp [req-b1b52991-302e-426b-9934-7617a7a8e23d 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Done building host file /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-28 03:23:41.820 19219 DEBUG neutron.agent.linux.utils [req-b1b52991-302e-426b-9934-7617a7a8e23d 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27944'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:41.844 19219 DEBUG neutron.agent.linux.dhcp [req-b1b52991-302e-426b-9934-7617a7a8e23d 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Reloading allocations for network: 13c94b81-58d3-4646-b469-39c74e1b872e reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-28 03:23:41.846 19219 DEBUG neutron.agent.linux.utils [req-b1b52991-302e-426b-9934-7617a7a8e23d 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-4', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:41.871 19219 DEBUG neutron.agent.linux.utils [req-b1b52991-302e-426b-9934-7617a7a8e23d 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-6', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:41.897 19219 DEBUG neutron.agent.linux.utils [req-b1b52991-302e-426b-9934-7617a7a8e23d 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:23:41.898 19219 DEBUG neutron.agent.linux.utils [req-b1b52991-302e-426b-9934-7617a7a8e23d 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:23:41.899 19219 DEBUG neutron.agent.linux.external_process [req-b1b52991-302e-426b-9934-7617a7a8e23d 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] No process started for 13c94b81-58d3-4646-b469-39c74e1b872e disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-28 03:23:41.901 19219 DEBUG oslo_concurrency.lockutils [req-b1b52991-302e-426b-9934-7617a7a8e23d 495dda5720eb49bd9e204b5f94df7ea0 d1812a4a358f413d963f4743db11b81f - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:23:44.512 19219 DEBUG oslo_concurrency.lockutils [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:23:44.513 19219 INFO neutron.agent.dhcp.agent [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-28T01:23:40Z, description=, device_id=460cb1b1-3a30-4fd3-b6a1-1297a2ddab08, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6ed2183a-68cb-4948-a7f8-adef908f6ca6', u'ip_address': u'11.235.1.7'}], id=4c7cc62b-ee0a-4d90-bc23-c8ccfd1531e4, mac_address=fa:16:3e:93:7c:1b, name=, network_id=13c94b81-58d3-4646-b469-39c74e1b872e, port_security_enabled=True, project_id=d1812a4a358f413d963f4743db11b81f, revision_number=4, security_groups=[u'ebbd80ef-9432-47b6-b001-b2448e141ec1'], status=DOWN, tags=[], tenant_id=d1812a4a358f413d963f4743db11b81f, updated_at=2018-06-28T01:23:43Z
2018-06-28 03:23:44.515 19219 DEBUG neutron.agent.dhcp.agent [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Calling driver for network: 13c94b81-58d3-4646-b469-39c74e1b872e action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-28 03:23:44.517 19219 DEBUG neutron.agent.linux.dhcp [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Building host file: /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-28 03:23:44.519 19219 DEBUG neutron.agent.linux.dhcp [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Done building host file /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-28 03:23:44.522 19219 DEBUG neutron.agent.linux.utils [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27944'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:44.535 19219 DEBUG neutron.agent.linux.dhcp [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Reloading allocations for network: 13c94b81-58d3-4646-b469-39c74e1b872e reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-28 03:23:44.536 19219 DEBUG neutron.agent.linux.utils [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-4', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:44.547 19219 DEBUG neutron.agent.linux.utils [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-6', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:44.556 19219 DEBUG neutron.agent.linux.utils [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:23:44.557 19219 DEBUG neutron.agent.linux.utils [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:23:44.557 19219 DEBUG neutron.agent.linux.external_process [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] No process started for 13c94b81-58d3-4646-b469-39c74e1b872e disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-28 03:23:44.558 19219 DEBUG oslo_concurrency.lockutils [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:23:45.210 19219 DEBUG oslo_concurrency.lockutils [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:23:45.211 19219 INFO neutron.agent.dhcp.agent [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-28T01:23:40Z, description=, device_id=460cb1b1-3a30-4fd3-b6a1-1297a2ddab08, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6ed2183a-68cb-4948-a7f8-adef908f6ca6', u'ip_address': u'11.235.1.7'}], id=4c7cc62b-ee0a-4d90-bc23-c8ccfd1531e4, mac_address=fa:16:3e:93:7c:1b, name=, network_id=13c94b81-58d3-4646-b469-39c74e1b872e, port_security_enabled=True, project_id=d1812a4a358f413d963f4743db11b81f, revision_number=5, security_groups=[u'ebbd80ef-9432-47b6-b001-b2448e141ec1'], status=DOWN, tags=[], tenant_id=d1812a4a358f413d963f4743db11b81f, updated_at=2018-06-28T01:23:44Z
2018-06-28 03:23:45.214 19219 DEBUG neutron.agent.dhcp.agent [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Calling driver for network: 13c94b81-58d3-4646-b469-39c74e1b872e action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-28 03:23:45.216 19219 DEBUG neutron.agent.linux.dhcp [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Building host file: /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-28 03:23:45.221 19219 DEBUG neutron.agent.linux.dhcp [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Done building host file /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-28 03:23:45.227 19219 DEBUG neutron.agent.linux.utils [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27944'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:45.246 19219 DEBUG neutron.agent.linux.dhcp [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Reloading allocations for network: 13c94b81-58d3-4646-b469-39c74e1b872e reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-28 03:23:45.249 19219 DEBUG neutron.agent.linux.utils [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-4', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:45.267 19219 DEBUG neutron.agent.linux.utils [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-6', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:23:45.289 19219 DEBUG neutron.agent.linux.utils [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:23:45.290 19219 DEBUG neutron.agent.linux.utils [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:23:45.291 19219 DEBUG neutron.agent.linux.external_process [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] No process started for 13c94b81-58d3-4646-b469-39c74e1b872e disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-28 03:23:45.292 19219 DEBUG oslo_concurrency.lockutils [req-f9613ff0-38d4-49e5-bdef-e6ceca4753c8 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:24:14.043 19219 DEBUG oslo_concurrency.lockutils [req-7dd55aaf-44a0-4ad1-bc9b-e33b862972b6 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:24:14.044 19219 DEBUG oslo_concurrency.lockutils [req-7dd55aaf-44a0-4ad1-bc9b-e33b862972b6 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Releasing semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:24:15.433 19219 DEBUG oslo_concurrency.lockutils [req-7dd55aaf-44a0-4ad1-bc9b-e33b862972b6 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:24:15.434 19219 DEBUG oslo_concurrency.lockutils [req-7dd55aaf-44a0-4ad1-bc9b-e33b862972b6 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Releasing semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:24:21.286 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 03:24:21.291 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 03:25:21.286 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 03:25:21.291 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 03:26:21.287 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 03:26:21.292 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 03:27:21.287 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 03:27:21.290 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 03:28:15.569 19219 DEBUG oslo_concurrency.lockutils [req-fe23656e-a13e-4471-89d1-366fb5725dc9 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:28:15.570 19219 DEBUG neutron.agent.dhcp.agent [req-fe23656e-a13e-4471-89d1-366fb5725dc9 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Calling driver for network: 13c94b81-58d3-4646-b469-39c74e1b872e action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-28 03:28:15.572 19219 DEBUG neutron.agent.linux.utils [req-fe23656e-a13e-4471-89d1-366fb5725dc9 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'dhcp_release', 'tap4e7d3607-d1', '11.235.1.7', 'fa:16:3e:93:7c:1b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:28:15.594 19219 DEBUG neutron.agent.linux.dhcp [req-fe23656e-a13e-4471-89d1-366fb5725dc9 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Building host file: /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-28 03:28:15.602 19219 DEBUG neutron.agent.linux.dhcp [req-fe23656e-a13e-4471-89d1-366fb5725dc9 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Done building host file /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-28 03:28:15.614 19219 DEBUG neutron.agent.linux.utils [req-fe23656e-a13e-4471-89d1-366fb5725dc9 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27944'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:28:15.624 19219 DEBUG neutron.agent.linux.dhcp [req-fe23656e-a13e-4471-89d1-366fb5725dc9 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Reloading allocations for network: 13c94b81-58d3-4646-b469-39c74e1b872e reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-28 03:28:15.626 19219 DEBUG neutron.agent.linux.utils [req-fe23656e-a13e-4471-89d1-366fb5725dc9 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-4', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:28:15.641 19219 DEBUG neutron.agent.linux.utils [req-fe23656e-a13e-4471-89d1-366fb5725dc9 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-6', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:28:15.654 19219 DEBUG neutron.agent.linux.utils [req-fe23656e-a13e-4471-89d1-366fb5725dc9 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:28:15.655 19219 DEBUG neutron.agent.linux.utils [req-fe23656e-a13e-4471-89d1-366fb5725dc9 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:28:15.655 19219 DEBUG neutron.agent.linux.external_process [req-fe23656e-a13e-4471-89d1-366fb5725dc9 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] No process started for 13c94b81-58d3-4646-b469-39c74e1b872e disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-28 03:28:15.656 19219 DEBUG oslo_concurrency.lockutils [req-fe23656e-a13e-4471-89d1-366fb5725dc9 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:28:21.287 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 03:28:21.290 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 03:29:13.654 19219 DEBUG oslo_concurrency.lockutils [req-cd090590-0537-43b0-ac9c-ad2c5c7a5952 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:29:13.657 19219 DEBUG neutron.agent.dhcp.agent [req-cd090590-0537-43b0-ac9c-ad2c5c7a5952 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Calling driver for network: 13c94b81-58d3-4646-b469-39c74e1b872e action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-28 03:29:13.660 19219 DEBUG neutron.agent.linux.utils [req-cd090590-0537-43b0-ac9c-ad2c5c7a5952 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'dhcp_release', 'tap4e7d3607-d1', '11.235.1.1', 'fa:16:3e:43:84:f7'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:29:13.691 19219 DEBUG neutron.agent.linux.dhcp [req-cd090590-0537-43b0-ac9c-ad2c5c7a5952 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Building host file: /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-28 03:29:13.696 19219 DEBUG neutron.agent.linux.dhcp [req-cd090590-0537-43b0-ac9c-ad2c5c7a5952 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Done building host file /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-28 03:29:13.701 19219 DEBUG neutron.agent.linux.utils [req-cd090590-0537-43b0-ac9c-ad2c5c7a5952 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['kill', '-HUP', '27944'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:29:13.718 19219 DEBUG neutron.agent.linux.dhcp [req-cd090590-0537-43b0-ac9c-ad2c5c7a5952 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Reloading allocations for network: 13c94b81-58d3-4646-b469-39c74e1b872e reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-28 03:29:13.720 19219 DEBUG neutron.agent.linux.utils [req-cd090590-0537-43b0-ac9c-ad2c5c7a5952 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-4', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:29:13.737 19219 DEBUG neutron.agent.linux.utils [req-cd090590-0537-43b0-ac9c-ad2c5c7a5952 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e', 'ip', '-6', 'route', 'list', 'dev', 'tap4e7d3607-d1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:29:13.753 19219 DEBUG neutron.agent.linux.utils [req-cd090590-0537-43b0-ac9c-ad2c5c7a5952 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:29:13.754 19219 DEBUG neutron.agent.linux.utils [req-cd090590-0537-43b0-ac9c-ad2c5c7a5952 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:29:13.755 19219 DEBUG neutron.agent.linux.external_process [req-cd090590-0537-43b0-ac9c-ad2c5c7a5952 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] No process started for 13c94b81-58d3-4646-b469-39c74e1b872e disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-28 03:29:13.757 19219 DEBUG oslo_concurrency.lockutils [req-cd090590-0537-43b0-ac9c-ad2c5c7a5952 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:29:21.288 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 03:29:21.293 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 03:29:43.409 19219 DEBUG oslo_concurrency.lockutils [req-31086fd4-d65d-42e9-b9c2-db7b1f8746d7 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:29:43.412 19219 DEBUG neutron.agent.dhcp.agent [req-31086fd4-d65d-42e9-b9c2-db7b1f8746d7 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Calling driver for network: 13c94b81-58d3-4646-b469-39c74e1b872e action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-28 03:29:43.416 19219 DEBUG neutron.agent.linux.utils [req-31086fd4-d65d-42e9-b9c2-db7b1f8746d7 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['kill', '-9', '27944'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:29:43.454 19219 DEBUG neutron.agent.linux.utils [req-31086fd4-d65d-42e9-b9c2-db7b1f8746d7 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-28 03:29:43.496 19219 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tap4e7d3607-d1) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-28 03:29:43.604 19219 DEBUG neutron.agent.linux.utils [req-de824bbb-2b6d-44a9-ab93-ee74913017a9 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:29:43.621 19219 DEBUG neutron.agent.linux.utils [req-de824bbb-2b6d-44a9-ab93-ee74913017a9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:29:43.638 19219 DEBUG oslo_concurrency.lockutils [req-de824bbb-2b6d-44a9-ab93-ee74913017a9 - - - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:29:54.877 19219 DEBUG oslo_concurrency.lockutils [req-978b1d74-7411-49b7-9003-63ec8eba2322 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:29:55.316 19219 DEBUG neutron.agent.linux.utils [req-9d77216d-21c3-44c4-b2d2-7ec512d97d0d - - - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:29:55.317 19219 DEBUG neutron.agent.linux.utils [req-9d77216d-21c3-44c4-b2d2-7ec512d97d0d - - - - -] Unable to access /var/lib/neutron/external/pids/13c94b81-58d3-4646-b469-39c74e1b872e.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:29:55.318 19219 DEBUG neutron.agent.linux.external_process [req-9d77216d-21c3-44c4-b2d2-7ec512d97d0d - - - - -] No process started for 13c94b81-58d3-4646-b469-39c74e1b872e disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-28 03:29:55.320 19219 DEBUG neutron.agent.dhcp.agent [req-9d77216d-21c3-44c4-b2d2-7ec512d97d0d - - - - -] Calling driver for network: 13c94b81-58d3-4646-b469-39c74e1b872e action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-28 03:29:55.323 19219 DEBUG neutron.agent.linux.utils [req-9d77216d-21c3-44c4-b2d2-7ec512d97d0d - - - - -] Unable to access /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:29:55.324 19219 DEBUG neutron.agent.linux.utils [req-9d77216d-21c3-44c4-b2d2-7ec512d97d0d - - - - -] Unable to access /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-28 03:29:55.325 19219 DEBUG neutron.agent.linux.external_process [req-9d77216d-21c3-44c4-b2d2-7ec512d97d0d - - - - -] No process started for 13c94b81-58d3-4646-b469-39c74e1b872e disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-28 03:29:55.326 19219 DEBUG neutron.agent.linux.dhcp [req-9d77216d-21c3-44c4-b2d2-7ec512d97d0d - - - - -] Impossibile accedere a /var/lib/neutron/dhcp/13c94b81-58d3-4646-b469-39c74e1b872e/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:268
2018-06-28 03:29:55.327 19219 DEBUG neutron.agent.linux.dhcp [req-9d77216d-21c3-44c4-b2d2-7ec512d97d0d - - - - -] No interface exists for network 13c94b81-58d3-4646-b469-39c74e1b872e destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1482
2018-06-28 03:29:55.371 19219 DEBUG neutron.agent.linux.utils [req-26ed4231-83d1-46dc-b26b-921757fbff2f - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-28 03:29:55.391 19219 DEBUG neutron.agent.linux.dhcp [req-26ed4231-83d1-46dc-b26b-921757fbff2f - - - - -] Namespace already deleted: qdhcp-13c94b81-58d3-4646-b469-39c74e1b872e _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:246
2018-06-28 03:29:55.394 19219 DEBUG oslo_concurrency.lockutils [req-26ed4231-83d1-46dc-b26b-921757fbff2f - - - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:29:56.177 19219 DEBUG neutron.agent.dhcp.agent [req-dc9d0def-dab4-4438-8891-2675a9a0aedc - - - - -] resync (13c94b81-58d3-4646-b469-39c74e1b872e): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:255
2018-06-28 03:29:56.179 19219 INFO neutron.agent.dhcp.agent [req-dc9d0def-dab4-4438-8891-2675a9a0aedc - - - - -] Synchronizing state
2018-06-28 03:29:57.643 19219 INFO neutron.agent.dhcp.agent [req-5509421c-5904-4db6-8c90-b1ebb7810764 - - - - -] All active networks have been fetched through RPC.
2018-06-28 03:29:57.647 19219 INFO neutron.agent.dhcp.agent [-] Starting network 13c94b81-58d3-4646-b469-39c74e1b872e dhcp configuration
2018-06-28 03:29:57.648 19219 INFO neutron.agent.dhcp.agent [-] Finished network 13c94b81-58d3-4646-b469-39c74e1b872e dhcp configuration
2018-06-28 03:29:57.649 19219 INFO neutron.agent.dhcp.agent [-] Starting network 6ca5b7c5-23c0-4771-ad3c-ae15aacd634c dhcp configuration
2018-06-28 03:29:57.651 19219 INFO neutron.agent.dhcp.agent [-] Finished network 6ca5b7c5-23c0-4771-ad3c-ae15aacd634c dhcp configuration
2018-06-28 03:29:57.652 19219 INFO neutron.agent.dhcp.agent [req-5509421c-5904-4db6-8c90-b1ebb7810764 - - - - -] Synchronizing state complete
2018-06-28 03:29:57.653 19219 DEBUG oslo_concurrency.lockutils [req-978b1d74-7411-49b7-9003-63ec8eba2322 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-28 03:29:57.655 19219 DEBUG oslo_concurrency.lockutils [req-978b1d74-7411-49b7-9003-63ec8eba2322 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Releasing semaphore "dhcp-agent-network-lock-13c94b81-58d3-4646-b469-39c74e1b872e" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-28 03:30:21.288 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 03:30:21.293 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.006s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 03:31:21.288 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 03:31:21.292 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 03:32:21.288 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 03:32:21.290 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 03:33:21.289 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 03:33:21.291 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 03:34:21.289 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 03:34:21.291 19219 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
